unused-markup.md

```diff
@@ -0,0 +1,131 @@
+ <!-- <tr>
```

Comment:
what is the purpose of this?

Reply:
stuff I might need again later on, I can track it outside of git, i'll remove this file.